openshift/forms-flow-ai-charts/charts/forms-flow-web/templates/configmap.yaml

```diff
@@ -41,8 +50,7 @@ data:
         "REACT_APP_DOCUMENT_SERVICE_URL": "${REACT_APP_DOCUMENT_SERVICE_URL}",
         "REACT_APP_DRAFT_ENABLED": "${REACT_APP_DRAFT_ENABLED}",
         "REACT_APP_DRAFT_POLLING_RATE": "${REACT_APP_DRAFT_POLLING_RATE}",
-        "REACT_APP_EXPORT_PDF_ENABLED": "${REACT_APP_EXPORT_PDF_ENABLED}",
-        "REACT_APP_CUSTOM_LOGOUT_URL": "${REACT_APP_CUSTOM_LOGOUT_URL}",
-        "REACT_APP_CUSTOM_MAP_URL": "${REACT_APP_CUSTOM_MAP_URL}"
+        "REACT_APP_EXPORT_PDF_ENABLED": "${REACT_APP_EXPORT_PDF_ENABLED}"
```

Comment:
We will eventually need to re-add 

```
        "REACT_APP_CUSTOM_LOGOUT_URL": "${REACT_APP_CUSTOM_LOGOUT_URL}",
        "REACT_APP_CUSTOM_MAP_URL": "${REACT_APP_CUSTOM_MAP_URL}"
```

But it's fine if we do it later.  

I wound up having to manually add them when testing in dev.